specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -9049,6 +9165,10 @@
           "description": "The Id of the machine where the provider is getting added.",
           "type": "string"
         },
+        "biosId": {
```

Comment:
This (along with all the other property changes in this PR) needs to go in a new api-version

Reply:
All the added properties are optional, so it shouldn't impact client usage and require a new api-version.

Reply:
If you think this qualifies for an exception to the normal breaking change definition you can request an exemption via the process here:  https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes

Reply:
Adding optional properties is an evolutionary change that requires a new API version. Please see the MS API Guidelines here for details: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-changes-that-require-a-version-change

Reply:
These properties are specific to V2A RCM provider which is not released to customers yet. So this shouldn't qualify as evolutionary change.

Reply:
Please request an exception through the breaking change process linked above.  The swagger spec is for a stable api-version from 2018